Automattic/jetpack
# Widget Visibility: ensure we find the correct site type

#### Changes proposed in this Pull Request:

﻿We previously relied on `wpcom`, which unfortunately won't work here; it does end up being defined even on Jetpack sites, because of the additional scripts from WordPress.com that are enqueued in the admin.

This was reported here: p8oabR-Nm-p2#comment-5896
#### Jetpack product discussion

* N/A

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* You'll want to test this on wpcom simple as well as Jetpack
* In Jetpack > settings, ensure the Widget visibility feature is active.
* go to Appearance > Widgets
* Add a block
* in the right block sidebar, scroll down and open the "Advanced" panel.
* Add a visibility rule
* in the options, you should see the option to filter by user role, or to check whether the user is logged in or not.
    * On WordPress.com, those options should not appear.
